d2renderers/d2svg/d2svg.go

```diff
@@ -1594,6 +1594,22 @@ func EmbedFonts(buf *bytes.Buffer, diagramHash, source string, fontFamily *d2fon
 }`,
 	)
 
+	appendOnTrigger(
+		buf,
+		source,
+		[]string{
+			`text-link`,
+		},
+		`
+.text-link {
+	fill: blue;
+}
+		
+.text-link:visited {
+	fill: #1a0dab;
```

Comment:
do we have the explicitly set this? If you leave it unset, would it just work or no? If it's required, I think just the default "purple" is best.

Reply:
Maybe it's just a browser issue on my end but these are my results (:visited is set to true in dev tools).

| No class styling | Only `.text-link` | Original code |
|--------|--------|--------|
| <img width="472" alt="Screenshot 2024-05-31 at 00 05 50" src="https://github.com/terrastruct/d2/assets/63919918/3c19730e-0765-4013-ae51-ec9c00d7c0fe"> | <img width="327" alt="Screenshot 2024-05-31 at 00 06 09" src="https://github.com/terrastruct/d2/assets/63919918/dac1e901-6eae-4ab0-bb4f-e9b6e04242ce"> | <img width="156" alt="Screenshot 2024-05-31 at 00 09 33" src="https://github.com/terrastruct/d2/assets/63919918/c96c6470-5f8e-4e34-9cdc-f7d56573732b"> |